server/controllers/finance/reports/unpaid_invoice_payments/index.js

```diff
@@ -66,6 +66,7 @@ async function getUnbalancedInvoices(options) {
   const params = [
     new Date(options.dateFrom),
     new Date(options.dateTo),
+    +options.currencyId,
```

Comment:
This is cool, but I really worry about this being `NaN`.  What happens then?  What does MySQL convert it to?

It would be much better to convert this explicitly to something.